cdap-ui/app/cdap/components/DataPrep/Directives/Filter/index.js

```diff
@@ -225,7 +225,7 @@ export default class FilterDirective extends Component {
     return (
       <div>
         <div className="custom-condition-container">
-          <strong>{T.translate(`${PREFIX}.customconditionlabel`)}</strong>
+          <strong>{T.translate(`${PREFIX}.customconditionlabel`, {column: this.props.column})}</strong>
```

Comment:
if the label is only the column, then we can remove the entire `T.translate`. Just do `{this.props.column}`

Reply:
thought about it, but we wouldn't ever need this changed right? maybe not, we can worry aobut it later. removing.

Reply:
fixed